Assets/Mirror/Runtime/NetworkWriter.cs

```diff
@@ -26,120 +26,68 @@ public class NetworkWriter
     {
         public const int MaxStringLength = 1024 * 32;
 
-        // create writer immediately with it's own buffer so no one can mess with it and so that we can resize it.
-        // note: BinaryWriter allocates too much, so we only use a MemoryStream
-        // => 1500 bytes by default because on average, most packets will be <= MTU
-        byte[] buffer = new byte[1500];
+        // fixed size buffer so we can pool it and don't need runtime resizing.
```

Comment:
this comment is un-needed, readonly already implies it doesn't change at runtime. Maybe this comment should explain why it is readonly instead?